frontend_app_logbook/src/component/LogbookEntryEditor.jsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+import {
+  DateInput,
+  IconButton,
+  TextInput,
+  TinyEditor
+} from 'tracim_frontend_lib'
+
+function LogbookEntryEditor (props) {
+  const { entry } = props
+
+  const [title, setTitle] = useState(entry.title || '')
+  const [description, setDescription] = useState(entry.description || 'This is a description')
+  const [bgColor, setBgColor] = useState(entry.bgColor || '#e8e8e8')
+  const [deadline, setDeadline] = useState(entry.deadline || getCurrentDateTime())
+  const [freeInput, setFreeInput] = useState(entry.freeInput || '')
+
+  function getCurrentDateTime () {
```

Comment:
is this function required?
couldn't you do the same using date-fns that is available in tracim? (https://date-fns.org/v3.3.1/docs/formatISO)

or https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString

It seems to do the same

Reply:
fixed. More details in code:

```
  // NOTE - M.L. - 2024-02-28 - This function is required due to the very specific format requested
  //  by the 'datetime-local' input type
  function getCurrentDateTime () {
    return new Date().toISOString().replaceAll('Z', '')
  }
```

Reply:
fixed